modules/apps/commerce/commerce-test-util/src/main/java/com/liferay/commerce/test/util/CommerceTaxTestUtil.java

```diff
@@ -33,7 +33,7 @@ public static CommerceTaxMethod addCommerceByAddressTaxMethod(
 		String commerceTaxMethodEngineKey = "by-address";
 
 		return CommerceTaxMethodLocalServiceUtil.addCommerceTaxMethod(
-			userId, commerceChannelGroupId,
+			commerceChannelGroupId, userId,
```

Comment:
mmm, does this compile? Because the true at the end is strange, the active param now is at the beginning